mobile/build.gradle.kts

```diff
@@ -204,6 +204,7 @@ dependencies {
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
 
     implementation(Libs.CORE_KTX)
+    implementation(Libs.APP_STARTUP)
```

Comment:
can this be debugImplementation?

Reply:
Hmm won't it fail in release app? Please note this is not `Startup Macrobenchmark` library. I believe, we need `AppStartup` for release variant, as well, no?